frontend/static/js/pages/play.js

```diff
@@ -209,10 +232,16 @@ let app = new Vue({
             if (this.path.length == 0 || this.path[this.path.length - 1]["article"] != page) {
                 // Update path
                 let timeElapsed = (Date.now() - this.startTime) / 1000;
+		
+		        if (this.isPenaltyMode) {
+                    this.penaltyTime = (this.path.length) * 20;
```

Comment:
You can check if `page === this.endArticle` so we don't penalize reaching the goal